rx_signal_processing/rx_dsp_chain.cu

```diff
@@ -238,129 +218,124 @@ int main(int argc, char **argv){
     );
 
 
+
+    auto complex_taps = filters.get_filter_taps();
```

Comment:
could the mixing to bandpass happen inside this get_filter_taps function? Potential for bug in this function if bandpass_filters hasn't been filled.

Reply:
I don't really like mixing those two separate behaviours together like that. But I did make two methods for getting both the mixed and unmixed taps so its clearer to whats being returned.

Reply:
that's great